vendor/github.com/docker/licensing/storage.go

```diff
@@ -87,18 +87,23 @@ func (c *client) LoadLocalLicense(ctx context.Context, clnt WrappedDockerClient)
 		licenseData, err = readLicenseFromHost(ctx, info.DockerRootDir)
 	} else {
 		// Load the latest license index
-		latestVersion, err := getLatestNamedConfig(clnt, licenseNamePrefix)
+		var latestVersion int
+		latestVersion, err = getLatestNamedConfig(clnt, licenseNamePrefix)
 		if err != nil {
 			if strings.Contains(err.Error(), "not a swarm manager.") {
```

Comment:
Please don't do string matching here; check if the node is a manger before calling this,  e.g., as is done here; https://github.com/docker/cli/blob/ae1618713f83e7da07317d579d0675f578de22fa/cli/command/stack/swarm/deploy.go#L49-L62

Not a blocker for this PR, but we should address this

Reply:
@mason-fish do you have any updates in the queue where you could absorb this or should I get a PR posted?

Reply:
Sure I can make the change once I'm through refactoring the error lib